### PR TITLE
Add VS Code extension for inline completion

### PR DESCRIPTION
**Pull request by akkashk**
Thank you for putting together this amazing tool, I spent some time writing a little VS Code extension that talks to the proxy server and fetches inline completion suggestions. It's been really fun playing around with fauxpilot with this extension and thought I would share the code in case it's useful for other people.

### CODE REVIEW

**Comment by moyix** [comment]
This is great (and I'm glad to hear you got FauxPilot working and have been using it :))! How does this compare to https://github.com/Venthe/vscode-fauxpilot ? Would it make sense to combine efforts there?

File: extension/.vscodeignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.vscode/**
```
Comment (leemgs) [inline extension/.vscodeignore:1]: What for?

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,8 @@ models/*
 .vscode
 .vscode/*
 .history
+
+# VidualStudioCode extension
+extension/.vscode
+extension/node_modules
+extension/dist
```
Comment (leemgs) [inline .gitignore:15]: Please fix the "new line" issue.

File: extension/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (leemgs) [inline extension/.eslintrc.json:1]: I think that you need to depict  why this commit is to be merged by appending commit description enough.

**Comment by leemgs** [comment]
To all reviewers,

After the reviewer's comment, there is no PR update for a long time.
If there is still no proper response from the submitter, we would like this PR to be closed now.

**Comment by thakkarparth007** [comment]
Closing this due to inactivity. Please feel free to reopen if there are updates.

**Pull request closed without merging**